### PR TITLE
[SPARK-15435][SQL] Append Command to all commands

### PR DESCRIPTION
**Pull request by rxin**
## What changes were proposed in this pull request?

We started this convention to append Command suffix to all SQL commands. However, not all commands follow that convention. This patch adds Command suffix to all RunnableCommands.
## How was this patch tested?

Updated test cases to reflect the renames.

### CODE REVIEW

File: sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala [hunk 1]
```diff
@@ -459,7 +459,7 @@ case class AlterTableSetLocation(
 }
 
 
-private[sql] object DDLUtils {
+object DDLUtils {
```
Comment (hvanhovell) [inline sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala]: Where do we need increased visibility?

**Comment by hvanhovell** [comment]
One minor comment. LGTM pending Jenkins.

File: sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala (hunk 1 above)
Comment (rxin) [inline sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala]: everything in catalyst was meant to be private so we didn't have any private[sql]

Comment (andrewor14) [inline sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala]: that's fine

**Comment by andrewor14** [comment]
LGTM2

File: sql/core/src/main/scala/org/apache/spark/sql/execution/command/AnalyzeTableCommand.scala [hunk 2]
```diff
@@ -106,6 +106,6 @@ case class AnalyzeTable(tableName: String) extends RunnableCommand {
   }
 }
 
-object AnalyzeTable {
+object AnalyzeTableCommand {
```
Comment (viirya) [inline sql/core/src/main/scala/org/apache/spark/sql/execution/command/AnalyzeTableCommand.scala]: nit: this object looks unnecessary.

Comment (rxin) [inline sql/core/src/main/scala/org/apache/spark/sql/execution/command/AnalyzeTableCommand.scala]: yup - it's been there though.

**Comment by rxin** [comment]
Merging in master. Thanks.

**Pull request closed without merging**